src/Hopper/Internal/LoweredCore/ClosureConvertedANF.hs

```diff
@@ -79,39 +78,9 @@ newtype CodeArity =
     CodeArity { getCodeArity :: Word64 }
   deriving(Eq,Ord,Read,Show,Typeable,Data,Generic)
 
-{-
-subtle issue with typed closure conversion:
-all the fields of the
-
-
--}
-
 -- whether the binder position is a variable, wild card,
 -- live/dead, type/runtime rep info?
 
---- kill these stubs later
---type Relevance = ()
-type TypeCC = () -- TODO FIXME, wire in the real type info? This will be Term???!
-data BinderInfoCC =
-      BinderInfoDataCC
-        { relevanceBICC :: Relevance -- if zero, during evaluation we need not pass around
-                                     --  BUT during NORMALIZATION, we probably do
-                                     --  so the normalizer WILL thread around irrelevant values
-                                     -- to properly support dependent type checking
-                                     -- as is proper, because a runtime irrelevant value
-                                     -- SHOULD be relevant during type checking, or
-                                     -- it has no reason to exist
-        , typeBICC :: TypeCC  -- at least for now, closure converted stuff may need a
```

Comment:
What happened to CC'ed stuff needing a "slightly different type system fragment"?